intersection_transit_maneuvering/include/intersection_transit_maneuvering/call_interface.hpp

```diff
@@ -23,7 +23,7 @@
 class CallInterface
 {
     public:
-        virtual void call(carma_planning_msgs::srv::PlanTrajectory::Request::SharedPtr req, carma_planning_msgs::srv::PlanTrajectory::Response::SharedPtr resp) = 0;
+        virtual void call(carma_planning_msgs::srv::PlanTrajectory::Request::SharedPtr req, carma_planning_msgs::srv::PlanTrajectory::Response::SharedPtr& resp) = 0;
```

Comment:
probably not as important, but shouldt the req be also passed by reference?

Reply:
Req is not being changed, so we probably shouldn't pass them by reference